wsi_deid/web_client/views/ItemView.js

```diff
@@ -308,6 +316,66 @@ wrap(ItemView, 'render', function (render) {
         window.setTimeout(() => resizeRedactBackground(elem), 1000);
     };
 
+    const addAuxImageMaps = (settings, redactList) => {
+        this.$el.find('.g-widget-metadata-container.auximage .g-widget-auximage').each((idx, elem) => {
+            elem = $(elem);
+            let imageElem = elem.find('.g-widget-auximage-image');
+            elem.wrap($('<div class="wsi-deid-auximage-container"></div>'));
+            let keyname = elem.attr('auximage');
+            // return true to 'continue' the loop. use configuration to drive which images to skip map creation
+            if (!['label', 'macro'].includes(keyname)) {
+                return true;
+            }
+            if (keyname === 'macro' && settings.redact_macro_square) {
+                return true;
+            }
+            if (keyname === 'label' && settings.always_redact_label) {
+                return true;
+            }
+            let mapId = `${keyname}-map`;
+            let tilesPath = `item/${this.model.id}/tiles`;
+            let mapDiv = $(`<div id="${mapId}" class="wsi-deid-associated-image-map"></div>`);
+            mapDiv.attr('keyname', keyname);
+            elem.after(mapDiv);
+
+            restRequest({
+                url: `${tilesPath}/images/${keyname}/metadata`,
+                error: null
+            }).done((resp) => {
+                try {
+                    let params = window.geo.util.pixelCoordinateParams(
+                        `#${mapId}`, resp.sizeX, resp.sizeY, resp.sizeX, resp.sizeY);
+                    let imgH = imageElem.height();
+                    let imgW = imageElem.width();
+                    $(`#${mapId}`).width(imgW).height(imgH);
```

Comment:
Move this line setting the map size to before you call `pixelCoordinateParams`; some of the values returned by that call are based on the map size.  Otherwise, for some shapes and sizes of images, you might not be able to zoom all the way out or have other odd effects.